### PR TITLE
Moved cookie sync services from sandbox to connector.

### LINKED ISSUE #54: Move Magneto sync classes from sandbox here.

### PR DESCRIPTION
**Pull request by GrandLTU**
Closes #54

### CODE REVIEW

File: Magento/Cart.php [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+
+        return new RedirectResponse($url);
+    }
+
+    /**
+     * Url magento should redirect to after adding products.
+     *
+     * @return string
+     */
+    private function getBackUrl()
```
Comment (mindaugasbarysas) [inline Magento/Cart.php]: maybe ReturnUrl would be better?

File: Magento/Cart.php [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+    }
+
+    /**
+     * Ensures contents are valid.
+     *
+     * @param mixed $content
+     *
+     * @return array
+     */
+    private function cleanContent($content)
```
Comment (mindaugasbarysas) [inline Magento/Cart.php:174]: Does this validate or clean content?

File: Magento/Customer.php [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+    private $userData;
+
+    /**
+     * Appends current uri to a url.
+     *
+     * @param string $url
+     *
+     * @return string
+     */
+    private function appendBackUrl($url)
```
Comment (mindaugasbarysas) [inline Magento/Customer.php]: ReturnUrl?

File: Magento/Customer.php [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+    }
+
+    /**
+     * Ensures user data is valid.
+     *
+     * @param mixed $data
+     *
+     * @return ParameterBag
+     */
+    private function formatUserData($data)
```
Comment (mindaugasbarysas) [inline Magento/Customer.php:78]: Does this validate or format user data?

File: Magento/Cart.php (hunk 2 above)
Comment (GrandLTU) [inline Magento/Cart.php:174]: Cleans.

File: Tests/Unit/Magento/CartTest.php [hunk 5]
```diff
@@ -0,0 +1,329 @@
...
+
+/**
+ * Tests Magento Cart.
+ */
+class CartTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * Tests setters and getters.
+     */
+    public function testSettersGetters()
```
Comment (mindaugasbarysas) [inline Tests/Unit/Magento/CartTest.php:31]: I believe we have a test class for getters and setters. Why wasn't it used here?

File: Tests/Unit/Magento/CartTest.php [hunk 6]
```diff
@@ -0,0 +1,329 @@
...
+     * Tests getCheckoutUrl.
+     */
+    public function testGetCheckoutUrl()
+    {
+        $instance = $this->getCart();
+        $this->assertEquals('testing.url' . Cart::CHECKOUT_PATH, $instance->getCheckoutUrl());
+    }
+
+    /**
+     * Provider for cookie tes.
```
Comment (mindaugasbarysas) [inline Tests/Unit/Magento/CartTest.php]: test. :)

File: Magento/Customer.php (hunk 4 above)
Comment (GrandLTU) [inline Magento/Customer.php:78]: Formats/cleans.

File: Tests/Unit/Magento/CartTest.php (hunk 5 above)
Comment (GrandLTU) [inline Tests/Unit/Magento/CartTest.php:31]: Where is it located?

Comment (mindaugasbarysas) [inline Tests/Unit/Magento/CartTest.php:31]: ONGR\ConnectionsBundle\Tests\Unit\Entity\AbstractEntityTest

File: Magento/Cart.php (hunk 2 above)
Comment (mindaugasbarysas) [inline Magento/Cart.php:174]: Please amend the doc comment accordingly, then :)

File: Magento/Customer.php (hunk 4 above)
Comment (mindaugasbarysas) [inline Magento/Customer.php:78]: Doc comment should reflect that :)

File: Tests/Unit/Magento/CartTest.php (hunk 5 above)
Comment (GrandLTU) [inline Tests/Unit/Magento/CartTest.php:31]: It will not do as it does not work with interfaces and in cases when constructors should be disabled.

Comment (mindaugasbarysas) [inline Tests/Unit/Magento/CartTest.php:31]: Disregard my comment then.

**Pull request merged**